### PR TITLE
Enhancement and Bug Fixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Added CICD tag to search the "Continuous Delivery" solution pack
- Match the sequence of "Setup the Development Environment" task with "Setup the Source Control for Production Environment" task
- Remove Attachment dependecy to create the Export Tempalte
- Added "Content Development" Team
- Added Content and Setting Export templates in Export Template recordset
- Modified some comments
- Modified some Step names

### CODE REVIEW

File: modules/change_management/detail-layout.json [hunk 1]
```diff
@@ -218,34 +219,34 @@
         ],
         "actionButtons": [
             {
-                "id": "\/api\/3\/workflows\/8e709eb0-7c8b-434d-928f-e26eda75f2b1",
```
Comment (cs-pooja-singh) [inline modules/change_management/detail-layout.json:221]: why these id changes? did you delete and create new playbooks?

File: modules/people/mmd.json [hunk 2]
```diff
@@ -47,7 +46,7 @@
             "identifier": null,
             "unique": false,
             "recommend": false,
-            "uuid": "a7bbf1ba-b7e1-4c6e-b149-d63fada06b3f",
+            "skipSerialization": false,
```
Comment (cs-pooja-singh) [inline modules/people/mmd.json:49]: we can remove both mmd.json from the diff I believe. there is no change

File: playbooks/02 - Use Case - CICD/Check Playbook is Already Running.json [hunk 3]
```diff
@@ -13,41 +13,32 @@
         "playbook_name"
     ],
     "synchronous": false,
-    "collection": "\/api\/3\/workflow_collections\/4c58fed3-9e16-451a-9528-0be2b71acda4",
+    "collection": "\/api\/3\/workflow_collections\/36f63e54-01b1-45f7-bd33-c53dba44b797",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Check Playbook is Already Running.json]: why is the collection id itself changed? may cause issues during SP upgrade

Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Check Playbook is Already Running.json]: done

**Review by cs-pooja-singh — APPROVED** [review]

**Pull request merged**